osf/utils/machines.py

```diff
@@ -153,8 +154,9 @@ def notify_edit_comment(self, ev):
 
     def notify_withdraw(self, ev):
         context = self.get_context()
+        context['ever_public'] = self.machineable.ever_public
         try:
-            preprint_request_action = PreprintRequestAction.objects.get(target__id=self.machineable.id,
+            preprint_request_action = PreprintRequestAction.objects.get(target__target__id=self.machineable.id,
```

Comment:
`taget__id` is the `PrepreintRequest`'s id while `self.machineable.id` the `Preprint`'s id. Change it to `target__target__id`.